[DOCS-1956] Update docs for hashed API keys (#2018)
wandb/docs: models/integrations/fastai.mdx

---
title: fastai
---

import ApiKeyCreateStreamlined from "/snippets/en/_includes/api-key-create-streamlined.mdx";

> If you're using **fastai** to train your models, W&B has an easy integration using the `WandbCallback`. Explore the details in [interactive docs with examples →](https://app.wandb.ai/borisd13/demo_config/reports/Visualize-track-compare-Fastai-models--Vmlldzo4MzAyNA)

Reviewer: ```suggestion
You can integrate **fastai** with W&B using the `WandbCallback` class. Check out these [interactive docs with examples](https://app.wandb.ai/borisd13/demo_config/reports/Visualize-track-compare-Fastai-models--Vmlldzo4MzAyNA) for more details.
```

I'm wary of calling anything "easy"
